extensions/amp-story/1.0/amp-story-interactive.css

```diff
@@ -17,3 +17,30 @@
 .i-amphtml-story-interactive-container {
   font-family: 'Poppins', sans-serif !important;
 }
+
+.i-amphtml-story-interactive-container {
+	background: var(--interactive-prompt-background-color) !important;
+	border-radius: 32px !important;
+}
+
+.i-amphtml-story-interactive-prompt-container {
+	display: flex !important;
+	justify-items: center !important;
+	padding: 20px !important;
+	color: var(--interactive-prompt-text-color) !important;
+}
+
+.i-amphtml-story-interactive-prompt {
+	margin: 0px !important;
+	overflow : hidden !important;
+	text-overflow: ellipsis !important;
+	display: -webkit-box !important;
+	-webkit-line-clamp: 4 !important;
+	-webkit-box-orient: vertical !important;
+	visibility: visible !important;
+	font-size: 28px !important;
+	line-height: 36px !important;
+	max-height: 112px !important;
+	font-weight: bold !important;
+	text-align: var(--interactive-prompt-alignment) !important;
+}
```

Comment:
All these styles are shared for the container and prompt, moved them here so that they can be used with the binary polls